apps/src/redux/instructions.js

```diff
@@ -156,19 +164,54 @@ export const setHasAuthoredHints = hasAuthoredHints => ({
 // HELPERS
 
 /**
+ * Given instructions that look something like
+ *   '[pufferpig] <b>Puffer Pigs</b> roam around slowly<br/>'
+ * Replaces [pufferpig] with the appropriate image html.
+ * In most cases, no substitutions will be necessary and this method will just
+ * return the passed in htmlText. Substitutions currently only exist for star wars.
+ * @param {string} htmlText
+ * @param {Object.<string, string>} [substitutions] Dictionary strings (keys) to
+ *   replacement values.
+ */
+export const substituteInstructionImages = (htmlText, substitutions) => {
+  if (!htmlText) {
+    return htmlText;
+  }
+
+  for (let prop in substitutions) {
+    const value = substitutions[prop];
+    const substitutionHtml = (
+      '<span class="instructionsImageContainer">' +
+        '<img src="' + value + '" class="instructionsImage"/>' +
+      '</span>'
```

Comment:
This would be a good place for an ES2015 template string.  Also, could we name `value` something more descriptive like `imageUrl`?